bundles/core/src/main/java/com/adobe/cq/wcm/core/components/internal/link/LinkHandler.java

```diff
@@ -101,22 +105,26 @@
             return Optional.empty();
         }
         String linkURL = getPageLinkURL(page);
-        return Optional.of(new LinkImpl<>(linkURL, null, page));
+        return Optional.of(new LinkImpl<>(linkURL, null, null, null, page));
     }
 
     /**
-     * Builds a link with the given Link URL and target.
+     * Builds a link with the given Link URL, target, accessibility label, title.
      * @param linkURL Link URL
      * @param target Target
+     * @param linkAccessibilityLabel Link Accessibility Label
+     * @param linkTitleAttribute Link Title Attribute
      *
      * @return {@link Optional} of  {@link Link<Page>}
      */
     @NotNull
-    public Optional<Link<Page>> getLink(@Nullable String linkURL, @Nullable String target) {
+    public Optional<Link<Page>> getLink(@Nullable String linkURL, @Nullable String target, @Nullable String linkAccessibilityLabel, @Nullable String linkTitleAttribute) {
```

Comment:
To avoid changes in files unrelated to this issue, we should still provide a method with the old signature.